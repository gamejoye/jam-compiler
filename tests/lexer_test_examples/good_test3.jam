let s: String = "Hello, \"gamejoye!\"";