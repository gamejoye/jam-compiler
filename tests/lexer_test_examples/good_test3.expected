LET
OBJECTID
:
TYPEID
=
STR_CONST
;